EDDiscovery/EDDiscovery.csproj

```diff
@@ -51,6 +51,7 @@
     <DefineConstants>TRACE</DefineConstants>
```

Comment:
All this wierdness is showing up because I tweaked the `Startup Object` property to go through `program.cs`. Because I renamed `Form1_Load` to `EDDiscoveryForm_Load`.